README.md

```diff
@@ -1 +1,27 @@
-# smallrye-config
+# SmallRye Config
+
+
+SmallRye Config is an implementation of [Eclipse MicroProfile Config][microprofile-config].
+
+[![Build Status](https://ci.wildfly.org/app/rest/builds/buildType:(id:MicroProfileJsr_WildflyMicroprofileConfig_MpConfig1xBranch)/statusIcon)](https://ci.wildfly.org/viewType.html?buildTypeId=MicroProfileJsr_WildflyMicroprofileConfig_MpConfig1xBranch&guest=1)
```

Comment:
I kept the link to WildFly CI that was used for wildfly-microprofile-config.

Should I also set it up for that project or had you in mind another CI infra?

Reply:
I was thinking of using Semaphore CI, as we'll eventually need one more jobs for each repo and will be easier if people outside RH can set them up as well

Reply:
I removed the link

Reply:
Thanks